src/components/User/OfflineOperations/Preissuance/components/PreissuanceForm.vue

```diff
@@ -260,7 +256,7 @@ export default {
         this.fileInfo = []
         Bus.success('preissuance-form.submitted-successfully')
       } catch (error) {
-        ErrorHandler.process(error)
+        ErrorHandler.process(error, 'preissuance-form.exceeded-max-preissuance')
```

Comment:
You always show one error message. If error message displaying incorrectly, maybe error contains incorrect msg

Reply:
changed